lib/nerves_hub/deployments/orchestrator.ex

```diff
@@ -49,6 +51,11 @@ defmodule NervesHub.Deployments.Orchestrator do
   As devices update and reconnect, the new orchestrator is told that the update
   was successful, and the process is repeated.
   """
+  @decorate with_span("Deployments.Orchestrator.trigger_update")
```

Comment:
```suggestion
  @decorate with_span("Deployments.Orchestrator.trigger_update#noop")
```
This will allow us to easily see why one trigger is faster than others. We might want to remove this `with_span` entirely, but for now I think the `#noop` will help.